plugins/plugin-svelte/plugin.js

```diff
@@ -45,17 +45,10 @@ module.exports = function plugin(snowpackConfig, pluginOptions = {}) {
           })
         ).code;
       }
-      // COMPILE
-      const ssrOptions = {};
-      if (isSSR) {
-        ssrOptions.generate = 'ssr';
-        ssrOptions.hydratable = true;
-        ssrOptions.css = true;
```

Comment:
The idea here was that CSS would be applied via JS directly in Svelte, so that you wouldn't need to handle our `*.css.proxy.js` proxy CSS files in SSR mode. Would love to hear your rationale behind removing this

Reply:
The `css` option doesn't do anything in SSR mode. In DOM mode it causes the generated code to include the styles in the JS so that you don't need to figure out how to extract and serve a `.css` file — bit of a hack and not really recommended.

Any SSR solution *will* need to extract CSS, and either serve it as a file, consumed via a `<link>`, or embedding it in the rendered HTML:

```js
const { html, css } = App.render(props);

return `<!doctype html>
<head>
  <style}${css.code}</style>
</head>
<body>
  ${html}
</body>`;
```